buildroot-external/kernel/v6.6.y/device-support-pci.config

```diff
@@ -39,3 +39,10 @@ CONFIG_SND_HDA_CODEC_HDMI=m
 # NVMe SSD support
 CONFIG_NVME_CORE=y
 CONFIG_BLK_DEV_NVME=y
+
+# Mellanox ConnectX-3 NIC support
+CONFIG_MLX4_EN=m
+CONFIG_MLX4_EN_DCB=y
+CONFIG_MLX4_CORE=m
```

Comment:
This is redundant, it's selected by `CONFIG_MLX4_EN` and as it has no description in the Kconfig, setting it has no effect.
```suggestion
```